lib/msal-common/src/url/UrlString.ts

```diff
@@ -139,8 +139,7 @@ export class UrlString {
             Protocol: match[1],
             HostNameAndPort: match[4],
             AbsolutePath: match[5],
-            QueryString: match[7],
-            Hash: match[9]
+            QueryString: match[7]
```

Comment:
is `Hash` no longer needed?

Reply:
I had added it initially because my original implementation was going to call the navigate function with each component separated. The new interface will return the full url.

Reply:
We can add it in a separate PR if needed since it is missing. Just wanted to contain the changes here to what's needed for this feature.